rest/rest-sqs-testing-amazon-java-sdk/src/test/scala/org/elasticmq/rest/sqs/FifoDeduplicationTests.scala

```diff
@@ -0,0 +1,232 @@
+package org.elasticmq.rest.sqs
+
+import com.amazonaws.services.sqs.model.{
+  CreateQueueRequest,
+  MessageAttributeValue,
+  ReceiveMessageRequest,
+  SendMessageRequest
+}
+import org.elasticmq.rest.sqs.model.RedrivePolicy
+import org.elasticmq.rest.sqs.model.RedrivePolicyJson.format
+import org.scalatest.OptionValues
+import org.scalatest.matchers.should.Matchers
+import spray.json.enrichAny
+
+import scala.collection.JavaConverters.mapAsJavaMapConverter
+import scala.jdk.CollectionConverters.CollectionHasAsScala
+
+class FifoDeduplicationTests extends SqsClientServerCommunication with Matchers with OptionValues {
```

Comment:
do we test that the dedup ids are removed after 5 minutes? (by shifting the `NowProvider`, of course)

Reply:
I have added a test for this scenario :)